game-core/src/test/java/games/strategy/engine/config/FilePropertyReaderTest.java

```diff
@@ -34,12 +34,7 @@ public void setup() throws Exception {
   }
 
   @Test
-  public void constructorWithPath_ShouldThrowExceptionWhenFileDoesNotExist() {
```

Comment:
This test was removed because it became a duplicate of the test below due to the removal of the `FilePropertyReader(File)` constructor in #3773.